modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -377,8 +377,8 @@ FeatureFloodCount::writeCSVFile(const std::string file_name, const std::vector<T
 {
   if (processor_id() == 0)
   {
-    // typdef makes subsequent code easier to read...
-    typedef std::map<std::string, MooseSharedPointer<std::ofstream> >::iterator iterator_t;
+    // alias declaration
+    using iterator_t = std::map<std::string, std::unique_ptr<std::ofstream> >::iterator;
```

Comment:
I'm being informed that GoT is on. I guess I'd better quit this and go see if <INSERT FAVORITE CHARACTER NAME> dies tonight.